### ISSUE TITLE
App restart during Harvest: what happens?

### ISSUE THREAD

**Issue by JPrevost** [MITLibraries/scoap3hub#73]
If we restart the app during a harvest (env var change or deploy will do that on Heroku without warning), will the harvest lose it's progress or will it continue where it left off?

**Comment by JPrevost** [comment]
I think we know what will happen, this ticket should probably be re-worded to ask what we could/should do about it.

The gist of what happens is that the Items are all created effectively at the start of the harvest so we can assume they are created for most intents here. What takes real time, is the Cataloging. If a server restarts after Phase 1 (Item creation), all of the incomplete Items will show up in the Workbench under `Items with no Topics`.

What is likely a good feature to consider, would be to attempt to re-catalog all of those Items in that state as a manual SysAdmin feature. This would allow for relatively easy recovery from mass failures part way through a Harvest. If that seems reasonably, I can open a ticket for something like that as a Post Launch feature and close this ticket. There may be other options though...

**Comment by JPrevost** [comment]
We can handle this with one off reharvests via `Item with No Topics` SysAdmin page for now and if a more robust option needs to be developed it can be opened as a new ticket in the future.
